components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -62,24 +62,25 @@ import {
 import { Popover, PopoverContent, PopoverTrigger } from "../Popover/popover";
 import { Calendar } from "../Calendar/calendar";
 import CardRepoList, { RepoList } from "../CardRepoList/card-repo-list";
+import DevToSocialImg from "../DevToSocialImage.tsx/dev-to-social-img";
```

Comment:
Quick attention to the component directory name

```suggestion
import DevToSocialImg from "../DevToSocialImage/dev-to-social-img";
```

Reply:
Removing .tsx gives me eslint errors :/

Reply:
🤦DIdn't notice. Thanks